pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -505,6 +505,65 @@ spec:
       workspaces:
         - name: data
           workspace: release-workspace
+    - name: update-component-sbom
+      when:
+        - input: "$(tasks.collect-atlas-params.results.secretName)"
+          operator: notin
+          values: [""]
+        - input: "$(tasks.collect-atlas-params.results.updateComponentSBOM)"
+          operator: in
+          values: ["true"]
+      taskRef:
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/update-component-sbom/update-component-sbom.yaml
+        resolver: git
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      params:
+        - name: dataJsonPath
+          value: "$(tasks.collect-data.results.data)"
+        - name: downloadedSbomPath
+          value: "$(tasks.push-rpm-data-to-pyxis.results.sbomPath)"
+      runAfter:
```

Comment:
This will only run after `collect-atlas-params` because of the `when` condition. Can you also add it here? The reason we list the dependencies explicitly is that without it, the UI tends to show the graph incorrectly.

Reply:
Updated